refactor(curriculum): update RPG project to accommodate new intro pyramid project (#54435)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa264d23cdaa45a20efada.md

---
id: 62aa264d23cdaa45a20efada
title: Step 162
challengeType: 0
dashedName: step-162
---

# --description--

> After your `numbers` array, create a `while` loop. Your `while` loop should run while `numbers.length` is less than `10`.

Reviewer: ```suggestion
After your `numbers` array, create a `while` loop that runs as long as `numbers.length` is less than `10`.
```
